samples/__tests__/e2e.mediation.test.ts

```diff
@@ -5,46 +5,59 @@ import { sleep, toBeConnectedWith, waitForBasicMessage, waitForMediationRecord }
 import indy from 'indy-sdk'
 import logger from '../../src/__tests__/logger'
 import { MediationState } from '../../src'
+import { AgentConfig } from '../../src/agent/AgentConfig'
+import { IndyWallet } from '../../src/wallet/IndyWallet'
+import { Wallet } from '../../src/wallet/Wallet'
 
 expect.extend({ toBeConnectedWith })
 
 const aliceConfig: InitConfig = {
   label: 'e2e Alice',
-  mediatorUrl: 'http://localhost:4001',
+  host: 'http://localhost',
+  port: 3001,
   walletConfig: { id: 'e2e-alice' },
   walletCredentials: { key: '00000000000000000000000000000Test01' },
   autoAcceptConnections: true,
-  logger: logger,
   indy,
+  logger: logger,
 }
 
 const bobConfig: InitConfig = {
   label: 'e2e Bob',
-  mediatorUrl: 'http://localhost:4002',
+  host: 'http://localhost',
+  port: 3002,
   walletConfig: { id: 'e2e-bob' },
   walletCredentials: { key: '00000000000000000000000000000Test02' },
   autoAcceptConnections: true,
-  logger: logger,
   indy,
+  logger: logger,
 }
 
 const mediatorConfig: InitConfig = {
   label: 'e2e ned',
-  mediatorUrl: 'http://localhost:4003',
+  host: 'http://localhost',
+  port: 3003,
   walletConfig: { id: 'e2e-ned' },
   walletCredentials: { key: '00000000000000000000000000000Test03' },
   autoAcceptConnections: true,
-  logger: logger,
   indy,
+  logger: logger,
 }
 
-
 describe('with mediator', () => {
   let aliceAgent: Agent
   let bobAgent: Agent
   let mediator: Agent
   let aliceAtAliceBobId: string
+  /*let aliceConfig: AgentConfig
+  let aliceWallet: Wallet
 
+  beforeAll(async () => {
+    aliceConfig = new AgentConfig(aliceConfig)
+    aliceWallet = new IndyWallet(aliceConfig)
+    await aliceWallet.init()
+  })
+*/
```

Comment:
Should we remove this?

Reply:
It will need to be uncommented or removed, so yes. e2e.mediation.test.ts is under construction, that code is copied pasted from e2e.tests.ts from agent end to end testing.